### ISSUE TITLE
Unable to detect NVDA on windows.

### ISSUE THREAD

**Issue by Leos-Khai** [ndarilek/tts-rs#38]
I tried to run the examples provided in the repo. It was unable to detect NVDA and it was not able to default to SAPI. Running the 99 bottle example just loop with no output.
On windows 10.

**Comment by ndarilek** [comment]
I can't duplicate this with the latest NVDA.

Did you run `cargo clean` before running `cargo run --example 99bottles 
--features=tolk`?

**Comment by Leos-Khai** [comment]
> I can't duplicate this with the latest NVDA. Did you run `cargo clean` before running `cargo run --example 99bottles --features=tolk`?

Ok running with --features=tolk fix the NVDA issue, but when running without it still dont have sapi or a default tts for windows.

**Comment by ndarilek** [comment]
Note that SAPI isn't supported. The WinRT driver works with OneCore voices.

I also don't have issues with the 99bottles example and OneCore voices. 
Sorry I can't be more helpful.

**Comment by Leos-Khai** [comment]
> Note that SAPI isn't supported. The WinRT driver works with OneCore voices. I also don't have issues with the 99bottles example and OneCore voices. Sorry I can't be more helpful.

Alright, will mess with it more and try to see what  is the issue with it. Thanks for your assistance.
